low-level/files/opt/seagate/sspl/setup/sspl_reset.py

```diff
@@ -0,0 +1,158 @@
+import sys
+import errno
+import argparse
+import inspect
+import traceback
+import os
+import shutil
+import subprocess
+# Add the top level directories
+sys.path.insert(0, '/opt/seagate/cortx/sspl/low-level')
+
+from framework.base.sspl_constants import PRODUCT_FAMILY, file_store_config_path, enabled_products
+from files.opt.seagate.sspl.setup.sspl_setup import Cmd as SSPLSetup
+
+SSPL_CONFIGURED=f"/var/{PRODUCT_FAMILY}/sspl/sspl-configured"
+
+class SSPLResetCmd:
+
+    name="sspl-reset"
+
+    def __init__(self, args: list):
+        self.args = args
+        self._config()
+
+    def _config(self):
+        if os.path.exists(file_store_config_path):
+            with open(file_store_config_path, "r") as SSPL_CONF:
+                for line in SSPL_CONF:
+                    if 'data_path' in line:
+                        self.SSPL_DATA_DIR=line.split("=")[1]
+                    elif 'log_file_path' in line:
+                        self.IEM_FILE_PATH=line.split("=")[1]
+        else:
+            self.SSPL_DATA_DIR=f"/var/{PRODUCT_FAMILY}/sspl/data"
+            self.IEM_FILE_PATH=f"/var/{PRODUCT_FAMILY}/sspl/data/iem/iem_messages"
+
+    @staticmethod
+    def usage(prog: str):
+        """Print usage instructions"""
+
+        sys.stderr.write(
+            f"usage: {prog} [hard [-p <LDR_R2>]|soft]\n"
+            f"hard options:\n"
+            f"-p product to be configured\n")
+    
+    def process(self):
+        if not self.args:
+            self.usage(self.name)
+        i=0
+        while i < len(self.args):
+            try:
+                if self.args[i] == "hard":
+                    HardCmd(self.args).process()
+                elif self.args[i] == "soft":
+                    SoftCmd(self.args).process()
+            except (IndexError, ValueError):
+                self.usage(self.name)
+            i+=1
+
+    def del_file(self, filename):
+        if os.path.exists(filename):
+            os.remove(filename)
+
+
+class HardCmd(SSPLResetCmd):
+    """Hard reset Cmd"""
+    name = "hard"
+    user_present=False
+    
+    def __init__(self, args):
+        super().__init__(args)
+
+    def process(self):
+        print("Processing %s %s" %(self.name, self.args))
+        i = 0
+        while i < len(self.args):
+            try:
+                if self.args[i] == '-p':
+                    product = self.args[i+1]
+                    break
+            except (IndexError, ValueError):
+                self.usage()
+            i+=1
+        if product not in enabled_products:
+            self.usage(self.name)
+            return
+
+        # stop sspl service
+        SSPLSetup.systemctl_service_action(service='sspl-ll.service', action='stop')
```

Comment:
Couldn't find this function systemctl_service_action() in https://github.com/Seagate/cortx-monitor/blob/main/low-level/files/opt/seagate/sspl/setup/sspl_setup.py

We need to use dbus interface for service management

Reply:
i can find this in latest PR for post install

Reply:
@satish-seagate  has added the function in the latest PR:-
https://github.com/Seagate/cortx-monitor/pull/279

@cdeshmukh I have referred to that patch